.changeset/fresh-avocados-divide.md

```diff
@@ -0,0 +1,7 @@
+---
+"@neo4j/graphql": patch
+"@neo4j/introspector": patch
```

Comment:
We will get away without releasing the Introspector as this was not released following the accidental pinning. 🙂 

```suggestion
```